Radzen.Blazor/RadzenProfileMenu.razor.cs

```diff
@@ -42,6 +42,13 @@ protected override string GetComponentCssClass()
         [Parameter]
         public EventCallback<RadzenProfileMenuItem> Click { get; set; }
 
+        /// <summary>
+        /// Enable/Disable the "arrow down" icon
```

Comment:
Maybe the correct description should be Show/Hide instead

Reply:
Good call - updated!